sbnobj/Common/POTAccounting/classes.h

```diff
@@ -0,0 +1,9 @@
+#include <map>
+#include <string>
```

Comment:
I don't think `<map>` and `<string>` are needed.

Reply:
dropped